### ISSUE TITLE
CC1 and CC2 required for OTG?

### ISSUE THREAD

**Issue by etkaar** [ThomasVon2021/pikvm-usb-splitter#1]
Are CC1 and CC2 (and thus their resistors) even required? Isn't it that only D+ and D- would be required for OTG functionality?

**Comment by blikvm** [comment]
Yes, The CC pin is used to judge the direction of the power supply plug and the power supply end. If not, some plugs will have power supply compatibility problems.
